### ISSUE TITLE
[1.3.13] Project list is infinite

### ISSUE THREAD

**Issue by marktani** [graphcool/graphcool-cli#227]
After the last project, the first project comes again. Is this intended? If so, a visual cue that this is the list from the start would be helpful. (For example a line that only contains `-` characters).

**Comment by timsuchanek** [comment]
Yes, this is intended. We should indeed find a better UI for that
